lib/tests-from-pull-request.js

```diff
@@ -28,28 +28,35 @@ function getTestsFromPR(context) {
   }
 }
 
-function getTestNames(prPatch) {
+function getTestNames(prPatch, context) {
   const testNames = [];
   const parsedPatch = prPatch.split('\n');
-
-  parsedPatch.forEach((line) => {
-    if (line.startsWith('+')) {
-      line = removePrefixAndTrim(line);
-      const testName = getTestName(line);
-
-      testName ? testNames.push(testName) : null;
-    }
-  });
-
-  return testNames;
+  return getConfig(context)
+    .then(config => {
+      parsedPatch.forEach((line) => {
+        if (line.startsWith('+')) {
+          line = removePrefixAndTrim(line);
+          const testName = getTestName(line, config);
+
+          testName ? testNames.push(testName) : null;
+        }
+      });
+    })
+    .then(() => testNames);
 }
 
 function removePrefixAndTrim(line) {
   return line.slice(1).trim();
 }
 
-function getTestName(line) {
-  const getTestNameRegex = /it\((.*),/;
+function getConfig(context) {
+  return Promise.resolve(context.config('config.yml'));
+}
+
+function getTestName(line, config) {
+  const testFunctionName = config && config.tdd1tTestFunctionName ?
+    config.tdd1tTestFunctionName : 'it';
+  const getTestNameRegex = new RegExp(`${testFunctionName}\((.*),`);
```

Comment:
since this is a string now, you need to replace `\(` with `\\(` so it won't throw an error

Reply:
![screen shot 2019-02-07 at 5 55 44 pm](https://user-images.githubusercontent.com/10427304/52423877-9ff36180-2b01-11e9-92f3-2dd616d64ba5.png)